CHANGES.txt

```diff
@@ -27,6 +27,13 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
   From Ryan Saunders:
     - Fixed runtest.py failure on Windows caused by excessive escaping of the path to python.exe.
 
+  From Mats Wichmann:
+    - A list argument as the source to the Copy() action function is now
+      correctly handled by converting elements to string. Copy bails if
```

Comment:
maybe instead of `bails` will throw a `SCons.Errors.UserError` ?

Reply:
yes, ok - I'm going to change this, so it won't be a `UserError` anyway, but will adjust the wording.

Reply:
Gotcha. Too bad there's nothing which gets called at SConscript evaluation time for Actions which could validate this and fail then..